Source/Core/AudioCommon/OpenALStream.cpp

```diff
@@ -132,15 +132,21 @@ void OpenALStream::SoundLoop()
   Common::SetCurrentThreadName("Audio thread - openal");
 
   bool surround_capable = SConfig::GetInstance().bDPL2Decoder;
-#if defined(__APPLE__)
   bool float32_capable = false;
+  bool fixed32_capable = false;
+
+#if defined(__APPLE__)
   const ALenum AL_FORMAT_STEREO_FLOAT32 = 0;
+  const ALenum AL_FORMAT_STEREO32 = 0;
   // OS X does not have the alext AL_FORMAT_51CHN32 yet.
   surround_capable = false;
   const ALenum AL_FORMAT_51CHN32 = 0;
   const ALenum AL_FORMAT_51CHN16 = 0;
+#elif defined(_WIN32)
+  // Only Windows supports AL_FORMAT_STEREO32 alext for now.
+  const ALenum AL_FORMAT_STEREO32 = 0x1203;
```

Comment:
Sorry, I did not understand your request correctly. With `m_format` you mean member variable?
Maybe it is better to move this:
```
#if defined(__APPLE__)
  const ALenum AL_FORMAT_STEREO_FLOAT32 = 0;
  const ALenum AL_FORMAT_STEREO32 = 0;
  // OS X does not have the alext AL_FORMAT_51CHN32 yet.
  surround_capable = false;
  const ALenum AL_FORMAT_51CHN32 = 0;
  const ALenum AL_FORMAT_51CHN16 = 0;
#elif defined(_WIN32)
  // Only Windows supports AL_FORMAT_STEREO32 alext for now.
  const ALenum AL_FORMAT_STEREO32 = 0x1203;
#else
  const ALenum AL_FORMAT_STEREO32 = 0;
#endif
```
To the header file? And change it to something like this:
```
#if defined(__APPLE__)
// OS X does not have the alext AL_FORMAT_STEREO_FLOAT32, AL_FORMAT_STEREO32, AL_FORMAT_51CHN32 and AL_FORMAT_51CHN16 yet.
#define AL_FORMAT_STEREO_FLOAT32 = 0;
#define AL_FORMAT_STEREO32 = 0;
#define AL_FORMAT_51CHN32 = 0;
#define AL_FORMAT_51CHN16 = 0;
#elif defined(_WIN32)
// Only Windows supports AL_FORMAT_STEREO32 alext for now.
#define AL_FORMAT_STEREO32 = 0x1203;
#else
#define AL_FORMAT_STEREO32 = 0;
#endif
```

But keep this on OpenALStream.cpp:
```
#if defined(__APPLE__)
  surround_capable = false;
```